### DISCUSSION TITLE
Can Hexo remove "/" at the end of the url?

### DISCUSSION THREAD

**Discussion by annandadewapp** [hexojs/hexo#5320] [Question / Answer] [answered]
I just learned Hexo a few days ago, maybe this is a little unimportant but I really want to know the answer.

Before :

localhost:4000/categories/
localhost:4000/categories/example-category-a/
localhost:4000/categories/example-category-b/
localhost:4000/tags/
localhost:4000/tags/example-tag-a/
localhost:4000/tags/example-tag-b/

After:

localhost:4000/categories
localhost:4000/categories/example-category-a
localhost:4000/categories/example-category-b
localhost:4000/tags
localhost:4000/tags/example-tag-a
localhost:4000/tags/example-tag-b

I've used permalinks, but I can't get rid of the '/' symbox in each category/tags list

Is there a way to remove the '/" symbol at the end of each URL?

**Comment by uiolee** [comment, marked as answer]
Generally speaking, paths with a slash after them are considered folders, and paths without a slash are considered files.

https://hexo.io/docs/configuration#URL

if you want remove slash, try to set `permalink` ends with `.html` and turn off `trailing_html`

```diff yaml
# _config.yml

- permalink: :year/:month/:day/:title/
+ permalink: :year/:month/:day/:title.html

pretty_urls:
-  trailing_html: true # Set to false to remove trailing '.html' from permalinks
+  trailing_html: false # Set to false to remove trailing '.html' from permalinks

```
